irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -132,14 +126,34 @@ namespace iroha {
       return account;
     }
 
+    nonstd::optional<std::string> PostgresWsvQuery::getAccountDetail(
+        const std::string &account_id, const std::string &detail) {
+      pqxx::result result;
+      try {
+        result = transaction_.exec("SELECT data->>'" + detail + "' FROM account WHERE account_id = "
```

Comment:
isn't `detail` should be escaped? (quoted)

Reply:
isn't it quoted?